root/static/scripts/edit/URLCleanup.js

```diff
@@ -1990,6 +2072,45 @@ const CLEANUPS: CleanupEntries = {
       return {result: false, target: ERROR_TARGETS.URL};
     },
   },
+  'deviantart': {
+    match: [new RegExp('^(https?://)?(www\\.)?deviantart\\.com', 'i')],
+    restrict: [LINK_TYPES.artgallery],
+    clean: function (url) {
+      url = url.replace(/^(?:https?:\/\/)?(?:www\.)?deviantart\.com\/([^\/]+).*$/, 'https://www.deviantart.com/$1');
```

Comment:
It should also drop query and fragment:

```suggestion
      url = url.replace(/^(?:https?:\/\/)?(?:www\.)?deviantart\.com\/([^\/?#]+).*$/, 'https://www.deviantart.com/$1');
```